src/styles/styledComponents/QuontityCircle.tsx

```diff
@@ -1,13 +1,11 @@
 import { styled } from '@mui/system';
```

Comment:
pls rename to QuantityCircle.tsx (it was my typo in naming)

Reply:
fixed